pages/public_cloud/public_cloud_network_services/getting-started-09-creating-vrack-with-openstack/meta.yaml

```
@@ -0,0 +1,2 @@
+id: 24aab613-5d38-405b-aa7c-83951c6e64e1
+full_slug: public-cloud-network-vrack-openstack-api
```

```suggestion
full_slug: public-cloud-network-vrack-openstack-cli
```
wouldn't it be more logical?